crates/router/src/routes/app.rs

```diff
@@ -1639,6 +1639,19 @@ impl BusinessProfile {
     }
 }
 
+pub struct BusinessProfileNew;
+
+#[cfg(feature = "olap")]
+impl BusinessProfileNew {
+    pub fn server(state: AppState) -> Scope {
+        web::scope("/account/{account_id}/profile")
+            .app_data(web::Data::new(state))
+            .service(
```

Comment:
We can combine this with the original BusinessProfile routes right.

Reply:
No. I tried doing that, `{base_url}/account/{merchant_id}/connectors` was returning 404 not found.